src/components/ProductMilestoneCreateEditPage/ProductMilestoneCreateEditPage.tsx

```diff
@@ -224,21 +217,32 @@ export const ProductMilestoneCreateEditPage = ({ isEditPage = false }: IProductM
             </FormHelperText>
           }
         >
-          <InputGroup>
-            <InputGroupText>
+          <TextInputLoading
+            isRequired
+            type="text"
+            id={productMilestoneEntityAttributes.version.id}
+            name={productMilestoneEntityAttributes.version.id}
+            autoComplete="off"
+            {...register<string>(productMilestoneEntityAttributes.version.id, {
+              ...fieldConfigs.version,
+              validators: [
```

Comment:
None are overridden, since there are no other validators on `fieldConfig.version`. Typescript 
detects that `fieldConfig.version.validators` is nonexistent.

Reply:
So once someone else will add sync validator using standard way = he adds it into fieldConfig, what happens?

Reply:
Then async validator would override the ones declared in the config object.

Reply:
So I think it's not expected / desired behaviour and it should be solved now
1) config should be safely extended or
2) we could say sync + async are incompatible and only one of them can be used = implement basic checker throwing error if both of them are defined + brief TSDoc, this solution can be of course very limiting in the future

Reply:
1. it just cannot be done or I can't see a way to do that; anyway, it is the responsibility of the one adding the validator to check it and it would be bug easily findable bug
2. this is wrong, both can be used